code/game/machinery/doors/windowdoor.dm

```diff
@@ -173,8 +173,9 @@
 
 //When an object is thrown at the window
 /obj/machinery/door/window/hitby(AM as mob|obj)
-
-	..()
+	. = ..()
+	if(.)
+		return
```

Comment:
```
if(..())
  return
```

Reply:
But that would return 0 to a children of a children. Is that alright?

Reply:
Correct, I am idort